common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/MicrosoftTokenResponse.java

```diff
@@ -261,7 +261,11 @@ public void setClientInfo(final String clientInfo) {
      * @return The ext expires on to get.
      */
     public Date getExtExpiresOn() {
-        return mExtExpiresOn;
+        if (mExtExpiresOn == null){
```

Comment:
Here (and 277) are the only changes I made (for spotbugs).

Reply:
```suggestion
        if (mExtExpiresOn == null) {
```